exporter/kinesisexporter/README.md

```diff
@@ -1,3 +1,64 @@
 # Kinesis Exporter
 
-To be added.
\ No newline at end of file
+This exporter can be used to send metrics to Kinesis.
```

Comment:
Looked at the code and it seems to be traces not metrics

Reply:
yes. you were right.